src/question_files/independence_day.csv

```diff
@@ -4,4 +4,8 @@ Who signed the Declaration of Independence on July 4 1776?,John Hancock,green,in
 How many people signed the Declaration of Independence?,56,green,independence day
 Where is the oldest continuous 4th of July Parade?,"Bristol, RI",green,independence day
 What is the most popular form of celebration on Independence Day?,Having a cookout,green,independence day
-When is Independence Day celebrated in the United States?,July 4,green,independence day
\ No newline at end of file
+When is Independence Day celebrated in the United States?,July 4,green,independence day
+Name the Unalienable Rights.,Life, Liberty, and the Pursuit of Happiness,green,independence day

```

Comment:
have to be careful with commas. will be red incorrectly by csv parser. if you must include commas do so with a `" "` like in line 5